src/AbstractTween.js

```diff
@@ -1,8 +1,8 @@
-var Transition         = require('./Transition');
-var TransitionRelative = require('./TransitionRelative');
-
-var easingFunctions        = require('./easing');
-var interpolationFunctions = require('./interpolation');
+var Transition         		= require('./Transition');
+var TransitionCSS		= require('./TransitionCSS');
+var TransitionRelative 		= require('./TransitionRelative');
+var easingFunctions        	= require('./easing');
+var interpolationFunctions 	= require('./interpolation');

```

Comment:
Indentation is getting a bit out of whack :)
Luckily GitHub makes that very visible. Could you please check your editor's settings?